### PR TITLE
All payments view for user

### PR DESCRIPTION
**Pull request by gala377**

### CODE REVIEW

**Comment by Delebrith** [comment]
@gala377 please, fix failing build so we can merge this branch. @lychanl 's PR is waiting for it

File: apsi-enrollment-fe/src/app/modules/main/pages/all-payments/all-payments.component.spec.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { AllPaymentsComponent } from './all-payments.component';
+
+describe('AllPaymentsComponent', () => {
+  let component: AllPaymentsComponent;
+  let fixture: ComponentFixture<AllPaymentsComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
```
Comment (Grushenko) [inline apsi-enrollment-fe/src/app/modules/main/pages/all-payments/all-payments.component.spec.ts:10]: You should define mocked SpyObj for PaymentService here. As there are actually no tests here you can also remove that file for now (as an alternative for adding SpyObj).

File: apsi-enrollment-fe/src/app/core/model/pagination.model.ts [hunk 2]
```diff
@@ -5,3 +5,9 @@ export interface Page<T> {
   pageSize: number;
   items: T[];
 }
+
+export interface PageRequest {
+  pageNumber?: number;
+  pageSize?: number;
+  searchQuery?: string;
+}
```
Comment (Grushenko) [inline apsi-enrollment-fe/src/app/core/model/pagination.model.ts:13]: Missing new line

File: apsi-enrollment-fe/src/app/core/model/payment.model.ts [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import { Enrollment } from './enrollment.model';
+
+export interface Payment {
+    id: number;
+    orderId: number;
+    status: string;
+    enrollment: Enrollment; 
+}
```
Comment (Grushenko) [inline apsi-enrollment-fe/src/app/core/model/payment.model.ts:8]: Missing new line

File: apsi-enrollment-fe/src/app/modules/main/main.routing.ts [hunk 4]
```diff
@@ -3,7 +3,8 @@ import { RouterModule, Routes } from '@angular/router';
 import { MainComponent } from './main.component';
 import { AllEventsComponent } from './pages/all-events/all-events.component';
 import { HomeComponent } from './pages/home/home.component';
-import {NewEventComponent} from './pages/new-event/new-event.component';
+import { NewEventComponent } from './pages/new-event/new-event.component';
+import { AllPaymentsComponent } from './pages/all-payments/all-payments.component';
```
Comment (Grushenko) [inline apsi-enrollment-fe/src/app/modules/main/main.routing.ts:7]: There is linter rule for imports to be ordered alphabetically. You should use TSLint for that.

File: apsi-enrollment-fe/src/app/core/model/payment.model.ts (hunk 3 above)
Comment (gala377) [reply apsi-enrollment-fe/src/app/core/model/payment.model.ts:8]: done

File: apsi-enrollment-fe/src/app/core/model/pagination.model.ts (hunk 2 above)
Comment (gala377) [reply apsi-enrollment-fe/src/app/core/model/pagination.model.ts:13]: done

File: apsi-enrollment-fe/src/app/modules/main/pages/all-payments/all-payments.component.spec.ts (hunk 1 above)
Comment (gala377) [reply apsi-enrollment-fe/src/app/modules/main/pages/all-payments/all-payments.component.spec.ts:10]: removed

File: apsi-enrollment-fe/src/app/modules/main/main.routing.ts (hunk 4 above)
Comment (gala377) [reply apsi-enrollment-fe/src/app/modules/main/main.routing.ts:7]: done

**Pull request merged**